### ISSUE TITLE
adding materials to TeSS - allow ftp as well as http

### ISSUE THREAD

**Issue by orchid00** [ElixirTeSS/TeSS#570]
Suggestion to allow ftp as well as http in the URL field when adding new materials to TeSS.

Reference to the PLAZA materials.

Thanks!

**Comment by njall** [comment]
Do you have a link to PLAZA materials @orchid00 ?

**Comment by orchid00** [comment]
hi @njall how are you? :) always nice to hear from you. 

I might need to remind you, that we added the materials (last year).
https://tess.elixir-europe.org/materials/plaza-is-a-plant-oriented-online-resource-for-comparative-evolutionary-and-functional-genomics

When I was filling the form it will keep complaining that resource URL needs to be http, whereas the link I had was ftp.
After we talked you added the ftp in text. Not the best place but if people read the description they could find it.
So at that time there was still need to support links other than http. I guess it was only an issue of the string allowed on that field.
Back then you asked me to add this as issue. So here is the context in case it was forgotten ;) . 

Thanks for looking back at it.
So, for A _new_ link, I think the best person to ask will be Alex (or anyone from ELIXIR Belgium).
